src/libsystemd/sd-device/test-sd-device.c

```diff
@@ -499,6 +499,41 @@ TEST(sd_device_enumerator_add_match_parent) {
         }
 }
 
+TEST(sd_device_enumerator_add_all_parents) {
+        _cleanup_(sd_device_enumerator_unrefp) sd_device_enumerator *e = NULL;
+        unsigned devices_count_with_parents = 0;
+        unsigned devices_count_without_parents = 0;
```

Comment:
Please merge the lines:
```c
        unsigned devices_count_with_parents = 0, devices_count_without_parents = 0;
```
Or maybe move the declarations before the relevant loops?